test-cases/longevity/longevity-twcs-3h.yaml

```diff
@@ -23,4 +23,4 @@ space_node_threshold: 64424
 
 user_prefix: 'longevity-twcs-3h'
 
-post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 300;"
+post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test WITH tombstone_gc = {'mode':'immediate'};"
```

Comment:
@raphaelsc can you please advise - 
i'm not sure why setting mode 'immediate' in this non-TTL test.
Isn't this mode mainly relevant to twcs + TTL? wouldn't it be too-data-resurrection-risky user scenario?

Reply:
That is on me, I switched all tests, not sure if they are both relevant.